tools/test-proxy/Azure.Sdk.Tools.TestProxy/Record.cs

```diff
@@ -39,9 +39,8 @@ public void Start()
         public void Stop()
         {
             string id = RecordingHandler.GetHeader(Request, "x-recording-id");
-            bool save = bool.Parse(RecordingHandler.GetHeader(Request, "x-recording-save"));
```

Comment:
Why is the `x-recording-save` header no longer needed?  Is it because file-based recordings use auto-save, and memory-based recordings cannot be saved?

Reply:
Should the `PerfTest` client stop sending the `x-recording-save` header?  I didn't see it in your changes to `PerfTest`.

Reply:
Yes. But it's low-pri as sending additional headers doesn't break anything.